docs/pages/aws-terraform-guide.mdx

```diff
@@ -425,15 +425,15 @@ To add users to the Teleport cluster, you will need to connect to a Teleport aut
 1 - Use the AWS cli to get the IP of the bastion server:
 
 ```bash
-$ export BASTION_IP=$(aws ec2 describe-instances --filters "Name=tag:TeleportCluster,Values=${TF_VAR_cluster_name},Name=tag:TeleportRole,Values=bastion" --query "Reservations[*].Instances[*].PublicIpAddress" --output text)
+$ export BASTION_IP=$(aws ec2 describe-instances --filters "Name=tag:TeleportCluster,Values=${TF_VAR_cluster_name}" "Name=tag:TeleportRole,Values=bastion" --query "Reservations[*].Instances[*].PublicIpAddress" --output text)
```

Comment:
Hmm this is a weird syntax (how does the command determine that the second filter is a --filters flag value and not a positional argument?) but it seems to be indeed correct according to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Filtering.html examples. @corkrean Have you verified it works?

Reply:
Yes, I have verified it.